chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,90 @@
+straightforward API type hints and 100% string coverage in jina framework, you can enjoy a pretty smooth development experience. However, for YAML configs, it doesn't help so much besides simple code highlight which could slow down your development speed as you have to navigate through the documentation of the code base in order to figure out which component
+or argument you really want to use.
+
+## Enable Intellisense for jina yaml configs in Pycharm:
+
+- First open Pycharm, click on the Menu and select Preference. 
+- In the left panel, search for JSON schema in the search box and select JSON schema mappings.
+- Next, in the right panel click + icon and add a new Schema.
+- Name: Name as per choice (for eg. jina) 
+- In the Schema file or URL write `api.jina.ai/schemas/latest.json` ( this ensures that your schema is always up to date with the new release).
+- You can also bind it to the specific Jina version or pre-release version.
+- In the Schema Version, select JSON Schema version 7.
+- Finally, add to File path pattern mapping: *.jaml to associate any files that end with JAML or Jina yaml extensions to the schema.
+- Click OK to apply the change.
+
+![ide-speed-up](intelli.png "Enabling Intellisense")
+
+## For Visual Studio Code developers:
+
+- First install the YAML extension from the RedHat.
+- Click on the sidebar and select Extension.
+- Search for YAML and then select *YAML support* from RedHat and then install it.
+- Open your IDE-wise `settings.json`, create a new entry:
+```
+“yaml.schemas”: {
+“https://api.jina.ai/schemas/latest.json”: [“/*.jina.yml, /*.jaml]
+}
+```
+- Finally save your `settings.json` to apply the change.
+
+![ide-speed-up](vse.png "VSCode Autocomplete")
+
+
+## Using the VSCode and PyCharm extensions
+
+- Step 1:
+Create a new `hello.jina.yml` file and the IDE successfully marks it as Jina file type.
+If it doesn't, you may want to manually select the schema you just created.
+
+- Step 2:
+Now type Jtype in the first line and you will see the IDE suggests you with flow or a list of Executors that jina contains.
+
+![ide-speed-up](pyc.png "Pycharm Autocomplete")
+
+*jtype* is a synonym for the bond mark. We recommend you to use `jtype` over the bond mark as
+it gives a cross-platform compatible yaml files.
+Now, the IDE immediately marks it as yellow as it is not a valid jina config file.
+
+- Step 3:
+Hover your mouse on it and it complains you haven't defined the required field version and parts.
+It let's it complete them as you write urlc's autocomplete kicks in to help you fill in the default
```

Comment:
Does the start of that sentence make sense?

Reply:
yes, since we have a pic below to describe the state